test/system/252-quadlet.bats

```diff
@@ -1865,4 +1865,312 @@ EOF
+    assert "$output" !~ "$quadlet_name" "list should not contain removed container"
+}
+
+# bats test_tags=distro-integration
+@test "quadlet verb - install with external file mount" {
+    # Create a test.txt file with content
+    local test_file=$PODMAN_TMPDIR/test.txt
+    echo "This is a test file for quadlet mount testing" > $test_file
+
+    # Create a quadlet directory for installation
+    local quadlet_dir=$PODMAN_TMPDIR/quadlet-mount-test
+    mkdir -p $quadlet_dir
+
+    # Create quadlet file that mounts the external test.txt file
+    local quadlet_file=$quadlet_dir/mount-test.container
+    cat > $quadlet_file <<EOF
+[Container]
+Image=$IMAGE
+Exec=sh -c "echo STARTED CONTAINER WITH MOUNT; cat /mounted/test.txt; trap 'exit' SIGTERM; while :; do sleep 0.1; done"
+Mount=type=bind,source=./test.txt,destination=/test.txt:z
+Notify=yes
+LogDriver=passthrough
+EOF
+
+    # Determine the install directory path based on rootless/root
+    local install_dir
+    if is_rootless; then
+        # For rootless: $XDG_CONFIG_HOME/containers/systemd or ~/.config/containers/systemd
+        local config_home=${XDG_CONFIG_HOME:-$HOME/.config}
+        install_dir="$config_home/containers/systemd"
+    else
+        # For root: /etc/containers/systemd
+        install_dir="/etc/containers/systemd"
+    fi
+
+    # Clean all quadlets
+    run_podman quadlet rm --all -f
+    assert $status -eq 0 "all quadlets should be cleaned"
+
+    # Test quadlet install with the directory containing the quadlet and test file
+    run_podman quadlet install $quadlet_dir $test_file
+    assert $status -eq 0 "quadlet install with external file mount should succeed"
+
+    # Verify the test.txt file exists in the install directory
+    [ -e "$install_dir/test.txt" ]
+    assert $status -eq 0 "test.txt file should exist in install directory $install_dir"
+
+    # Verify the content of the installed test.txt file
+    run cat "$install_dir/test.txt"
+    assert $status -eq 0 "should be able to read installed test.txt file"
+    assert "$output" == "This is a test file for quadlet mount testing" "installed test.txt should have correct content"
+
+    # Test quadlet list to verify the container was installed
+    run_podman quadlet list
+    assert $status -eq 0 "quadlet list should succeed"
+    assert "$output" =~ "mount-test.container" "list should contain mount-test.container"
+
+    # Test quadlet print to verify the configuration includes the volume mount
+    run_podman quadlet print mount-test.container
+    assert $status -eq 0 "quadlet print should succeed"
+    assert "$output" =~ "\[Container\]" "print should show container section"
+    assert "$output" =~ "Image=$IMAGE" "print should show correct image"
+    assert "$output" =~ "Mount=type=bind,source=./test.txt,destination=/test.txt:z" "print should show volume mount configuration"
+
+    # Test quadlet rm
+    run_podman quadlet rm mount-test.container
```

Comment:
Is this call expected to also remove the "$install_dir/test.txt"? If yes, it should be verified. If not, aren't we leaving leftovers?